ui/app/manager/src/pages/page-alarms.ts

```diff
@@ -0,0 +1,1133 @@
+                    height: fit-content; 
+                }
+
+                .hidden {
+                    display: none;
+                    margin: 0 !important;
+                }
+                
+                .breadcrumb-container {
+                    padding: 0 20px;
+                    width: calc(100% - 40px);
+                    max-width: 1360px;
+                    margin-top: 10px;
+                    display: flex;
+                    align-items: center;
+                }
+
+                .breadcrumb-clickable {
+                    cursor: pointer;
+                    color: ${unsafeCSS(DefaultColor4)};
+                }
+
+                .breadcrumb-arrow {
+                    margin: 0 5px -3px 5px;
+                    --or-icon-width: 16px;
+                    --or-icon-height: 16px;
+                }
+            `];
+    }
+
+    @property()
+    public config?: PageAlarmsConfig;
+
+    @property()
+    public realm?: string;
+    @state()
+    public alarm?: AlarmModel;
+    @state()
+    public creationState?: {
+        alarmModel: AlarmModel;
+    };
+
+    @state()
+    protected _alarms: AlarmModel[] = [];
+    @state()
+    protected _linkedAssets: Asset[] = [];
+    @state()
+    protected _linkedUsers: User[] = [];
+    @state()
+    protected _loadedUsers: User[] = [];
+
+    protected _loading: boolean = false;
+    protected _assign: boolean = false;
+    protected _userId?: string;
+
+    @state()
+    public severity?: AlarmSeverity;
+
+    @state()
+    public status?: AlarmStatus;
+
+    @state()
+    public hide: boolean = false;
+
+    @state()
+    public assign: boolean = false;
+
+    @state()
+    public allActive: boolean = true;
+
+    @state()
+    protected _data?: SentAlarm[];
+
+    @state()
+    protected _loadAlarmsPromise?: Promise<any>;
+
+    @state()
+    protected _saveAlarmPromise?: Promise<any>;
+
+    @state()
+    protected _loadUsersPromise?: Promise<any>;
+
+    @query("#table")
+    protected _tableElem!: HTMLDivElement;
+    protected _table?: OrMwcTable;
+    protected _refresh?: number;
+    protected _pageCount?: number;
+    protected _currentPage: number = 1;
+
+    @state()
+    protected _selected?: TableRow[];
+
+    get name(): string {
+        return "alarm.alarm_plural";
+    }
+
+    constructor(store: Store<AppStateKeyed>) {
+        super(store);
+        window.addEventListener('selectChanged',  (event : CustomEvent)=>  {
+            this._selected = event.detail;
+        });
+    }
+
+    public connectedCallback() {
+        super.connectedCallback();
+        this._loadData().then();
+    }
+
+    public disconnectedCallback() {
+        super.disconnectedCallback();
+        this.reset();
+        window.removeEventListener('selectChanged', (event : CustomEvent)=>  {
+            this._selected = event.detail;
+        });
+    }
+
+    protected updated(changedProperties: Map<string, any>) {
+        setTimeout(() => {
```

Comment:
`setTimeout()` should not be necessary here, and wouldn't behalve much differently since the `updated()` lifecycle method is executed after rendering the UI.